frontend/my-adopthive-app/src/NavBar/NavBar.jsx

```diff
@@ -0,0 +1,32 @@
+import Profile from "../Profile/Profile";
+import { useState, useContext } from "react";
+import { UserContext } from "../UserContext";
+import { PiSidebarDuotone } from 'react-icons/pi';
+import './NavBar.css';
+
+function Navbar(props) {
+    const [ showProfile, setShowProfile ] = useState(false);
+    const { user, updateUser } = useContext(UserContext);
+
+
+    function handleProfileDisplay() {
+        setShowProfile(!showProfile);
+    }
+
+    return (
+        <div className='navbar'>
+            <div onClick={props.toggleSideBar} className='toggle-btn'>
+                <PiSidebarDuotone />
+            </div>
+            <div className='navbar-title'>
+                <h1>AdoptHive</h1>
```

Comment:
A suggestion: Have a utils file that stores shared functionalities/strings (i.e. the string AdoptHive). I see that it is used in multiple places, so storing it in a utils could be beneficial (for example, if you want to change the name afterwards, you don't have to go through all the files and change it one by one).

Reply:
What if I remove the AdoptHive and create an header component where the title is stored and then call it in my App.jsx